.gitignore

```diff
@@ -27,7 +27,7 @@ bug*.pl
 /config.sh
 /makeaperl
 /makedepend
-/makedepend_file
```

Comment:
The first few characters of the commit message are wrong

Reply:
> The first few characters of the commit message are wrong

What is wrong in the commit message? Do you mean instead something wrong in the p.r. description that you have fixed?

Reply:
On closer examination, I was wrong.  The problem is that the commit message title is too long, so in displaying it, github folded it and added dots to the end of the truncated first line, and to the beginning of the second.  We often go over the official limit of 50 chars in the title, but not by so many as this.  I suggest

> Makefile.SH: Regen makedependfile when makedependfile.SH changed